awx_collection/README.md

```diff
@@ -32,33 +32,11 @@ Installing the `tar.gz` involves no special instructions.
 ## Running
 
 Non-deprecated modules in this collection have no Python requirements, but
-may require the official [AWX CLI](https://pypi.org/project/awxkit/)
+may require the AWX CLI
```

Comment:
If we are are referencing the CLI, in general, I think we would probably want to link to it?

Reply:
Not my change, from templating.